packages/cosmos_utils/lib/amount_formatter.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:intl/intl.dart';
+
+String formatAmount(double amount) => NumberFormat.compactCurrency(symbol: '\$').format(amount);
```

Comment:
let's make it a bit more sophisticated:
```suggestion
String formatAmount(double amount, {String locale = "en_US", String symbol = '\$'}) => amount >= 10000
    ? NumberFormat.compactCurrency(
        symbol: symbol,
        locale: locale,
      ).format(amount)
    : NumberFormat.currency(
        locale: locale,
        symbol: symbol,
      ).format(amount);

```

Reply:
✅